docs/reference/docs/update.asciidoc

```diff
@@ -139,13 +162,19 @@ the request was ignored.
 [source,js]
 --------------------------------------------------
 {
+   "_shards": {
+        "total": 0,
+        "successful": 0,
+        "failed": 0
+   },
    "_index": "test",
    "_type": "type1",
    "_id": "1",
-   "_version": 1,
+   "_version": 6,
    "result": noop
```

Comment:
That way you can say "by default updates that don't change anything detect that they don't change anything and return `"result": "noop"` like this:
<TESTRESPONSE snippet here>

You can disable this behavior by setting `"detect_noop": false` like this:
<CONSOLE snippet here>